worksModal.js

```diff
@@ -0,0 +1,183 @@
+    "Project6 text Ipsuz is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+  ],
+  projectImgMobile:
+  [
+    'assets/images/p1Mobile.png',
+    'assets/images/p2Mobile.png',
+    'assets/images/p1Mobile.png',
+    'assets/images/p2Mobile.png',
+    'assets/images/p1Mobile.png',
+    'assets/images/p2Mobile.png',
+  ],
+  projectImgDesktop:
+  [
+    'assets/images/p1Desktop.png',
+    'assets/images/p2Desktop.png',
+    'assets/images/p1Desktop.png',
+    'assets/images/p2Desktop.png',
+    'assets/images/p1Desktop.png',
+    'assets/images/p2Desktop.png',
+
+  ],
+  linkLive:
+  [
+    'https://gandradep.github.io/setupPortfolioMobileFirst/',
+    '#',
+    'https://gandradep.github.io/setupPortfolioMobileFirst/',
+    'https://gandradep.github.io/setupPortfolioMobileFirst/',
+    'https://gandradep.github.io/setupPortfolioMobileFirst/',
+    'https://gandradep.github.io/setupPortfolioMobileFirst/',
+  ],
+  linkSource:
+  [
+    'https://github.com/gandradep/setupPortfolioMobileFirst',
+    '#',
+    'https://github.com/gandradep/setupPortfolioMobileFirst',
+    'https://github.com/gandradep/setupPortfolioMobileFirst',
+    'https://github.com/gandradep/setupPortfolioMobileFirst',
+    'https://github.com/gandradep/setupPortfolioMobileFirst',
+  ],
+};
+
+const worksGrid = document.getElementById('worksGrid');
+
+for (let i = 0; i < worksInfo.title.length; i += 1) {
+  const article = document.createElement('article');
+  article.className = 'workPost borderRad12';
+  const divCard = document.createElement('div');
+  divCard.className = 'workInfo borderRad12';
+  const hgroup = document.createElement('hgroup');
+  hgroup.className = 'postColor';
+
+  const title = document.createElement('h2');
+  const textTitle = document.createTextNode(worksInfo.title[i]);
+  title.className = 'cardTitle';
+  title.appendChild(textTitle);
+
+  const techList = document.createElement('ul');
+  techList.className = 'postColor';
+
+  const x = worksInfo.tech[i];
+
+  for (let j = 0; j < x.length; j += 1) {
+    const li = document.createElement('li');
+    li.innerText = x[j];
+    techList.appendChild(li);
+  }
+
+  const button = document.createElement('a');
+  button.innerText = 'See Project';
+  button.className = 'button';
+  button.href = '#';
+  hgroup.appendChild(title);
+  divCard.appendChild(hgroup);
+  divCard.appendChild(techList);
+  divCard.appendChild(button);
+  article.appendChild(divCard);
+  worksGrid.appendChild(article);
+}
+
+/* create modal window */
+const modal = document.getElementById('modal');
```

Comment:
@addod19 thanks for the feedback, I was considering the same, but if you see the template model its a plain white X, I even scale the size up. What do you suggest?

Reply:
@addod19 at the end to keep the template design I just increase the font size of the X so it could be seen clearly, here is a picture.
![Screenshot (122)](https://user-images.githubusercontent.com/8645872/188862886-9190da8c-a1fe-474b-8847-9e2a00ecc457.jpg)